packages/plugin/src/theia.d.ts

```diff
@@ -176,6 +176,17 @@ declare module '@theia/plugin' {
          * invoked with.
          */
         arguments?: any[];
+
```

Comment:
This is for compatibility with vscode API. Shouldn't adapter code from vscode api to theia api be in place where we construct the vscode API object?

Reply:
This is for compatibility with vscode API and language server's Command